test/e2e/config/e2e_conf.yaml

```diff
@@ -149,6 +149,7 @@ variables:
   IMAGE_CHECKSUM_TYPE: "md5"
   IMAGE_USERNAME: "metal3"
   NODE_DRAIN_TIMEOUT: "0s"
+  INIT_WITH_KUBERNETES_VERSION: "v1.24.1"
```

Comment:
how is this specific version was chosen? based on `UPGRADED_K8S_VERSION`? do we need an extra variable at all?
Edit: I don't see it being used anywhere

Reply:
hmm for now we are using what ever capi would define in their e2e config file but we can override that from the environment variable

Reply:
in fact this will be used in the in the capi upgrade test since we are passing our e2e config as input and and the test look for a var with this name although it has same value as `UPGRADED_K8S_VERSION`

Reply:
Let's set it to `UPGRADED_K8S_VERSION` not to change it in two places in the future?